x/circuit/module.go

```diff
@@ -11,13 +11,17 @@ import (
 
 	"cosmossdk.io/core/appmodule"
 	appmodulev2 "cosmossdk.io/core/appmodule/v2"
+	"cosmossdk.io/core/transaction"
+	"cosmossdk.io/runtime/v2"
```

Comment:
i think @julienrbrt wanted to avoid this. Is there anyway to do it without the import?

Reply:
Yes! I don't know why CI doesn't run right now, but it would have blocked it.

Reply:
Looks like it isn't imported anymore 👍🏾